playgrounds/jquery/theme-example.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>jQuery app</title>
+    <link rel="stylesheet" href="../../packages/styles/lib/themes/dark-scoped.css" />
+    <link rel="stylesheet" href="../../packages/styles/lib/themes/light.css" />
+</head>
+
+<body>
+    <script src="../../packages/jquery/lib/bundle.debug.js" charset="utf-8"></script>
+    <span>Component styled by global theme (the last css link to the styles/lib/themes/*.css)</span>
+    <div id="cmp">
+        <script>
+            (function() {
+                new window.DevExpress.ui.dxRadioButton(document.getElementById('cmp'), { value: false, label: 'Check me' })
```

Comment:
Minor: Since this is an example, it is better to also pass checked or defaultChecked props and set value to some string instead of boolean. Otherwise, it looks a bit confusing, one can think that value sets checked state.

Reply:
Agree